custom/aaa/utils.py

```diff
@@ -46,3 +46,13 @@ def _explain_query(cls, method, domain, window_start, window_end):
     with connections['aaa-data'].cursor() as cursor:
         cursor.execute('explain ' + agg_query, agg_params)
         return cls.__name__ + method.__name__, cursor.fetchall()
+
+
+def get_location_model_for_ministry(ministry):
+    if ministry == 'MoHFW':
+        return AggVillage
+    elif ministry == 'MWCD':
+        return AggAwc
+
+    # This should be removed eventually once ministry is reliably being passed back from front end
```

Comment:
right now it looks like the user gets a no permissions error if ministry isn't set - should it instead use this fallback?

Reply:
no permissions is only for mobile users. web users have access to both (but for now i'm defaulting to village).

I created https://dimagi-dev.atlassian.net/browse/QA-207 for Lukasz to follow up on once he implements passing the hierarchy chosen back from the UI